code/game/gamemodes/cult/runes.dm

```diff
@@ -477,16 +477,14 @@ structure_check() searches for nearby cultist structures required for the invoca
 	..()
 	send_to_playing_players('sound/effects/dimensional_rend.ogg')
 	var/turf/T = get_turf(src)
+	SSticker.mode.eldergod = 0
 	sleep(40)
 	if(src)
 		color = "#FF0000"
 	SSticker.mode.eldergod = FALSE
 	deltimer(GLOB.blood_target_reset_timer)
```

Comment:
should probably move this into /obj/singularity/narsie/large/cult over the blood_target line